packages/cli/src/scripts/create.js

```diff
@@ -1,16 +1,26 @@
 import stdout from '../utils/stdout';
 import ControllerFor from '../models/network/ControllerFor';
+import promptPassword from '../utils/promptPassword'
 
 export default async function createProxy({ packageName, contractAlias, initMethod, initArgs, network, txParams = {}, force = false, networkFile = undefined }) {
   if (!contractAlias) throw Error('A contract alias must be provided to create a new proxy.')
   
   const controller = ControllerFor(network, txParams, networkFile)
   if (controller.isLib) throw Error('Cannot create a proxy for a library project')
-  
   await controller.checkContractDeployed(packageName, contractAlias, !force);
-  const proxy = await controller.createProxy(packageName, contractAlias, initMethod, initArgs);
-
+  let proxy
+  try {
+  proxy = await controller.createProxy(packageName, contractAlias, initMethod, initArgs);
+  } catch(e) {
+    if (e.message === "signer account is locked"){
+      await promptPassword(txParams.from ? txParams.from : web3.eth.accounts[0])
+      createProxy(...arguments)
```

Comment:
Infinite recursion is the intended behavior here i.e. if the user enters an incorrect password, we want to prompt them to enter it again until they input the correct password.

Reply:
I'm sorry, I should have been more specific. I meant that this could lead to an infinite recursion in some **non intented** cases. Anyway, I understand that infinite password prompt is being managed inside `promptPassword.js`, and not here. You should fix this behavior and also consider to ask for the password 3 times at the most.